### PR TITLE
38 create select

### PR DESCRIPTION
**Pull request by ArtemHard**

### CODE REVIEW

File: src/assets/icons/ArrowDown.tsx
Comment (ArtemHard) [inline src/assets/icons/ArrowDown.tsx:1]: this component already exist with name ChevronUp

File: src/components/ui/select/select.tsx [hunk 1]
```diff
@@ -1,3 +1,51 @@
-export const Select = () => {
-  return <div>Select</div>
+import { forwardRef } from 'react'
+
+import * as Select from '@radix-ui/react-select'
+import { SelectItemProps } from '@radix-ui/react-select'
+
+import { ArrowDown } from '../../../assets/icons'
```
Comment (ArtemHard) [inline src/components/ui/select/select.tsx:6]: replace on Import ChevroneUp

Comment (Vladimir-Kod) [reply src/components/ui/select/select.tsx:6]: done

File: src/assets/icons/ArrowDown.tsx
Comment (Vladimir-Kod) [reply src/assets/icons/ArrowDown.tsx:1]: done

**Review by ArtemHard — COMMENTED** [review]
ok, thanks #77

**Pull request merged**